### PR TITLE
docs: TKC-2704: add docs for audit logs

### PR DESCRIPTION
**Pull request by dejanzele**
This PR adds docs for audit logs

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/testkube-pro/articles/audit-logs.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Audit Logs
+
+**Testkube** records events into its internal Audit Log so **Organization Owners** and **Admins** can have insights how are the **Organization Members** using the product.
```
Comment (jfermi) [inline docs/testkube-pro/articles/audit-logs.md:3]:
```suggestion
**Testkube** records events into its internal Audit Log so **Organization Owners** and **Admins** can have insights into how the **Organization Members** are using the product.
```

File: docs/testkube-pro/articles/audit-logs.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**Audit Logs** can be accessed via the **Audit log** menu item in the **Organization Management** page.
+
+Events in the **Audit Log** can be filtered by date range of recorded events, by event type (i.e. **Environment Deleted**), by a specific **Environment** or by the subject/actor who initiated the event.
+
+In addition to querying, **Audit Log** can also be exported into **CSV** or **JSON** format.
+
+Default retention period for **Audit Log** events is **180 days**.
```
Comment (jfermi) [inline docs/testkube-pro/articles/audit-logs.md:11]:
```suggestion
The default retention period for **Audit Log** events is **180 days**.
```

File: docs/testkube-pro/articles/audit-logs.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Configuration
+
+In **Testkube On-Prem**, following settings can be configured during installation via Helm parameters:
```
Comment (jfermi) [inline docs/testkube-pro/articles/audit-logs.md:15]:
```suggestion
In **Testkube On-Prem**, the following settings can be configured during installation via Helm parameters:
```

**Review by jfermi — COMMENTED** [review]
A couple of editing suggestions.

File: docs/testkube-pro/articles/audit-logs.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+**Audit Logs** can also be exported for further analysis. Supported formats are:
+* **CSV**
+* **JSON**
+
+This makes it easy to integrate the audit log events with other reporting or analysis tools.
+
+![Export Audit Logs](../../img/audit-log-export.png "Export Audit Logs")
+
+## Retention period
```
Comment (jfermi) [inline docs/testkube-pro/articles/audit-logs.md:35]:
```suggestion
## Retention Period
```

**Review by jfermi — COMMENTED** [review]
Looks good! One small editing suggestion.

**Comment by dejanzele** [comment]
@jfermi addressed

**Review by jfermi — APPROVED** [review]

**Pull request merged**